app/src/main/java/de/westnordost/streetcomplete/quests/surface/DetailRoadSurfaceForm.kt

```diff
@@ -0,0 +1,100 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import androidx.appcompat.app.AlertDialog
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AImageListQuestAnswerFragment
+import de.westnordost.streetcomplete.quests.OtherAnswer
+import de.westnordost.streetcomplete.util.TextChangedWatcher
+import de.westnordost.streetcomplete.view.Item
+
+class DetailRoadSurfaceForm  : AImageListQuestAnswerFragment<String, DetailSurfaceAnswer>() {
+
+    override val items: List<Item<String>>
+        get() = if (osmElement!!.tags["surface"] == "paved")
+            (PAVED_SURFACES + UNPAVED_SURFACES + GROUND_SURFACES).toItems()
+        else
+            (UNPAVED_SURFACES + GROUND_SURFACES + PAVED_SURFACES).toItems()
+
+    override val itemsPerRow = 3
+
+    override fun onClickOk(selectedItems: List<String>) {
+        // must not happen in isInExplanationMode
+        // must contain exactly 1 item (maxSelectableItems is set to 1 and isFormComplete requires selection)
+        applyAnswer(SurfaceAnswer(selectedItems[0]))
```

Comment:
```suggestion
        applyAnswer(SurfaceAnswer(selectedItems.single()))
```
For cases like these, there is already something in the Kotlin standard library: `fun Collection.single()`